captum/optim/_param/image/images.py

```diff
@@ -212,6 +259,17 @@ def forward(self) -> torch.Tensor:
 class LaplacianImage(ImageParameterization):
```

Comment:
usually we document all non-underscored methods in the public classes. If we don't want those methods to be public we can underscore them. I see that we have quite some in `images.py`

Reply:
I've added some more documentation and added underscore to a bunch of the public class methods!